src/command_modules/azure-cli-appservice/HISTORY.rst

```diff
@@ -2,6 +2,12 @@
 
 Release History
 ===============
+0.2.5
++++++
+* az appserviceplan create now supports creating Windows Containers App Service Plans - Preview
+* az webapp create now supports creating and managing Windows Containers Apps - Preview
+* az webapp config set now supports --windows-fx-version for changing the container image for Windows Containers Apps - Preview
```

Comment:
suggest simplify the whole 3 lines to: `[preview] support webapps hosting on windows container`

Reply:
in the `_help.py`, add an example under `az webapp create` to make the new things more visible

Reply:
Thanks for the feedback @yugangw-msft  I was the one who asked that each of these changes be added to history.rst. Your suggested change is more concsise.